test/check-application

```diff
@@ -267,16 +267,13 @@ class TestApplication(testlib.MachineCase):
         b.click(".modal-dialog div .pf-c-alert .pf-c-button")
         b.wait_not_present(".modal-dialog div .alert")
 
-        # HACK: Disable checking for varlink error since it makes podman to coredump
-        # When hack removed, don't forget to also do the check for user containers
```

Comment:
before someone points this out, we run this part of test on both user container and system one as well.